DopplerFiles/Controllers/UploadFileRequest.cs

```diff
@@ -0,0 +1,9 @@
+namespace DopplerFiles.Controllers
+{
+    public class UploadFileRequest
+    {
+        public byte[] Content { get; set; }
+        public string PathFile { get; set; }
```

Comment:
Does it require serializing the file content in a JSON or you can simply uploadthe file?

Reply:
Serializing is required